docs/team/nrehiew.md

```diff
@@ -0,0 +1,27 @@
+# Project Overview 
+Connectify is a networking platform designed to help professionals efficiently manage their connections and enhance their networking experiences.
```

Comment:
Great description for our project! However, shall we follow the format given in the template markdown file?